typing_literal.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# ***************************************************************************
+# *                                                                         *
+# *   Copyright (c) 2020 - Suraj <dadralj18@gmail.com>                      *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "Literal typehint"
+__author__ = ""
+__url__ = "https://www.freecadweb.org"
+
+
+import typing
+
+
+# Please keep __all__ alphabetized within each category.
+__all__ = [
+    # Super-special typing primitives.
+    "Literal",
+]
+
+
+class _SpecialForm(typing._Final, typing._Immutable, _root=True):
```

Comment:
I tested it on Python 3.6.7 and getting error.

`typing._Final` and `typing._Immutable` attributes not found. 

I suggest to just comment Literal and OrderedDict typing from code.

Reply:
Later, we will just uncomment these things.

Reply:
Done suggested changes @amrit3701